silx/gui/plot/items/core.py

```diff
@@ -374,7 +377,7 @@ def pick(self, x, y):
 
 # Mix-in classes ##############################################################
 
-class ItemMixInBase(qt.QObject):
+class ItemMixInBase(object):
```

Comment:
That's because of some issues with diamond inheritance, right?

Reply:
Yes. It looks to work, but in the end you have double execution of some `__init__`, or Qt signals of instancies are not well initialized, that's tricky.